engine/src/flutter/shell/platform/windows/flutter_windows_display_monitor.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_FLUTTER_WINDOWS_DISPLAY_MONITOR_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_FLUTTER_WINDOWS_DISPLAY_MONITOR_H_
+
+#include <windows.h>
+#include "shell/platform/windows/windows_proc_table.h"
+
+namespace flutter {
+
+class FlutterWindowsEngine;
+class FlutterWindowsDisplayMonitor {
```

Comment:
Can we call this just `DisplayMonitor`? (The `FlutterWindows` prefix is a legacy naming convention when we used to have both UWP and win32 support, but we're moving away from it)